pkg/proc/proc_test.go

```diff
@@ -2082,6 +2124,9 @@ func TestIssue573(t *testing.T) {
 }
 
 func TestTestvariables2Prologue(t *testing.T) {
+	if runtime.GOARCH == "arm64" {
```

Comment:
How is this test failing exactly? We're just checking that we can detect the prologue correctly. It should work and if it isn't then that's probably the reason why several other tests aren't working.

Reply:
Yes, it can't skip func prologue correctly on arm64, it still get the start of func include `runtime.morestack` head. I will fix it next time, since it do not affect the debugging on native debug a lot. Let's skip it just for now.

Reply:
Prologue skipping does in fact affect debugging a lot, I'd rather not skip it.

Reply:
Okey

Reply:
Fixed. But why CI failed...

Reply:
Fixed.